src/scales/scale.timeseries.js

```diff
@@ -102,24 +129,15 @@ class TimeSeriesScale extends TimeScale {
 	}
 
 	/**
-	 * @protected
+	 * @param {number} pixel
+	 * @return {number}
 	 */
-	getLabelTimestamps() {
+	getValueForPixel(pixel) {
 		const me = this;
-		const timestamps = me._cache.labels || [];
-		let i, ilen;
-
-		if (timestamps.length) {
-			return timestamps;
-		}
-
-		const labels = me.getLabels();
-		for (i = 0, ilen = labels.length; i < ilen; ++i) {
-			timestamps.push(me.parse(labels[i]));
-		}
-
-		// We could assume labels are in order and unique - but let's not
-		return (me._cache.labels = timestamps);
+		const table = this._table;
+		const offsets = me._offsets;
```

Comment:
My main question about putting it in core is whether the radial scales have any concept of offset. I'm not sure it even makes sense for `time` really. I think it really only is used for `category` and `timeseries` because it's meant for use with the bar chart